src/__tests__/override-severity.test.ts

```diff
@@ -0,0 +1,60 @@
+import { DiagnosticSeverity } from '@stoplight/types';
+import testRule, { expectRulesetFileExists } from 'src/__tests__/__helpers__/helper.ts';
+
+describe('ruleset file', () => {
+  it('exists', () => expectRulesetFileExists());
+});
+
+testRule(['override-severity', 'must-use-https-protocol-only'], [
+  {
+    name: 'wrong x-api-type: override not applied',
+    document: `
+openapi: 3.0.0
+info:
+  title: Example
+  version: 1.0.0
+  x-api-type: not-pygeoapi
```

Comment:
x-api-type could be an enum

- standard
- pygepapi
- some-other-in-future

Reply:
This should be changed in https://github.com/ukhsa-collaboration/standards-api/commit/ec47653d598d9a80623f559b102949b0001b15e3